interface/drugs/add_edit_drug.php

```diff
@@ -617,18 +616,19 @@ function sel_related() {
 
 </table>
 <div class="btn-group">
-<input type='submit' class="btn btn-primary" name='form_save' value='<?php echo  $drug_id ? xla('Update') : xla('Add') ; ?>' />
-
-<?php if (AclMain::aclCheckCore('admin', 'super') && $drug_id) { ?>
-<input class="btn btn-danger" type='submit' name='form_delete' value='<?php echo xla('Delete'); ?>' />
-<?php } ?>
-<input type='button' class="btn btn-secondary" value='<?php echo xla('Cancel'); ?>' onclick='window.close()' />
+    <button type='submit' class="btn btn-primary btn-save" name='form_save'><?php echo  $drug_id ? xla('Update') : xla('Add') ; ?></button>
```

Comment:
When I click Add, the form is reset and nothing is saved.

Reply:
I think changing from input to button can cause issues; rec ensuring testing whenever change an input to button.

Reply:
OK, I updated the commit.